app/controllers/service_uploads_controller.rb

```diff
@@ -45,15 +45,9 @@ def create
 
         return unless date_ended
 
-        discontinued_service = DiscontinuedService.new(
-          service: service,
-          discontinued_at: date_ended,
-          recorded_by_educator: current_educator,
-        )
+        service.update_attributes(:discontinued_at => date_ended)
```

Comment:
I replace `DiscontinuedService.new` with an `update_attributes(:discontinued_at) here and everywhere else.